### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by FailSpy** [FailSpy/abliterator.py#4]

**Comment by sam99dave** [comment]
Hey @FailSpy, in the benchmarking and inference section you have used ortho. I'm unable to find the import to it in the readme, can you let me know on this?

**Comment by gpiat** [comment]
Using abliterator on Google Colab is finnicky. As it is a very popular tool, I think the following should be added to the readme:

> For Google Colab users:
> Avoid using condacolab if at all possible.
> Downgrade  GCSFS and then install abliterator from github whilst specifying the requirements file manually:
> `!pip install gcsfs==2024.9.0.post1`
> `!pip install "git+https://github.com/FailSpy/abliterator.git#egg=abliterator" -r https://raw.githubusercontent.com/FailSpy/abliterator/main/requirements.txt`

**Comment by gpiat** [comment]
> Hey [@FailSpy](https://github.com/FailSpy), in the benchmarking and inference section you have used ortho. I'm unable to find the import to it in the readme, can you let me know on this?

I don't think it would be imported: since ortho has an `apply_refusal_dir` method, I assume it should be `apply_refusal_dirs` (plural) and would be a ModelAbliterator object. Should it be `my_model` instead of `ortho`?

I can't find anything on `mse_harmless` either.
